graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017-present Kensho Technologies, LLC.
+import datetime
+import os
+import random
+import sys
+
+from ... import __version__
```

Comment:
Since `scripts` isn't supposed to be a package, you might have to read the version "manually" -- read the file then regex out the `__version__` variable's value. The `setup.py` already does this, follow its lead.